Editor/Properties/TeleportPropertyEditor.cs

```diff
@@ -0,0 +1,95 @@
+using System;
+using UnityEditor;
+using UnityEngine;
+using Innoactive.Creator.Core.Properties;
+using UnityEngine.XR.Interaction.Toolkit;
+
+namespace Innoactive.CreatorEditor.XRInteraction
+{
+    /// <summary>
+    /// Custom inspector for <see cref="TeleportProperty"/>, adding a button to automatically configure <see cref="Innoactive.Creator.XRInteraction.TeleportationAnchor"/>s.
+    /// </summary>
+    [CustomEditor(typeof(TeleportProperty)), CanEditMultipleObjects]
+    internal class TeleportPropertyEditor : Editor
+    {
+        private bool isNotSetup = true;
+        private const string AnchorPrefabName = "anchor";
+
+        private void OnEnable()
+        {
+            TeleportProperty teleportProperty = target as TeleportProperty;
+            TeleportationAnchor teleportAnchor = teleportProperty.GetComponent<TeleportationAnchor>();
+            
+            if (teleportProperty.transform.childCount != 0 && teleportAnchor.teleportAnchorTransform.name == AnchorPrefabName)
+            {
+                isNotSetup = false;
+            }
+        }
+
+        public override void OnInspectorGUI()
+        {
+            base.OnInspectorGUI();
+            EditorGUILayout.Space();
+
+            if (isNotSetup && GUILayout.Button("Set Default Teleportation Anchor"))
+            {
+                foreach (UnityEngine.Object targetObject in serializedObject.targetObjects)
+                {
+                    if (targetObject is TeleportProperty teleportationAnchor)
+                    {
+                        ConfigureDefaultTeleportationAnchor(teleportationAnchor);
+                    }
+                }
+            }
+        }
+        
+        private  void ConfigureDefaultTeleportationAnchor(TeleportProperty teleportationAnchor)
```

Comment:
Might be good to be able to overwrite this?

Reply:
Maybe, in general, it would be good to hook up custom actions when the fix button from the step inspector is hit.

I also thought about this (after the meeting with Sebastian), I'll say it is out of scope for this PR but we definitively should keep an eye on how to provide a way to do custom configurations.